02_configure_host.sh

```diff
@@ -124,7 +124,7 @@ if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
 fi
 
 # Add firewall rules to ensure the image caches can be reached on the host
-for PORT in 80 5000 ; do
+for PORT in 80 5000 ${LOCAL_REGISTRY_PORT} ; do
```

Comment:
5000 can be removed from this list as $LOCAL_REGISTRY_PORT==5000

Reply:
Done